fantasycourt.nl/content/programma_2025/artiesten/_index.nl.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Programma
```

Comment:
Hoewel de dropdown een goede toevoeging is, splits je nu de programma-pagina op in verschillende delen. Een dropdown is vrij vervelend navigeren op mobiele apparaten (grootste deel van bezoekers). Dan wil je er in principe doorheen kunnen (doom)scrollen.
Je kan vanuit een dropdown ook doorlinken naar een specifieke plek op een enkele pagina, dus ik zou de dropdown ook zeker houden